source/Camera/InfoUtilOther/Warmup/Warmup.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+This example shows how to warm up the camera.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+using HighResClock = std::chrono::high_resolution_clock;
+using Duration = std::chrono::nanoseconds;
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        auto warmupTime = std::chrono::minutes(10);
+        auto captureCycle = std::chrono::seconds(5);
+
+        std::cout << "Warm-up time: " << warmupTime.count() << " minutes" << std::endl;
+
+        const auto beforeWarmup = HighResClock::now();
+
+        const auto parameters = Zivid::CaptureAssistant::SuggestSettingsParameters{
+            Zivid::CaptureAssistant::SuggestSettingsParameters::AmbientLightFrequency::none,
+            Zivid::CaptureAssistant::SuggestSettingsParameters::MaxCaptureTime{ std::chrono::milliseconds{ 1000 } }
+        };
+        const auto settings = Zivid::CaptureAssistant::suggestSettings(camera, parameters);
+
+        while(HighResClock::now() - beforeWarmup < warmupTime)
+        {
+            const auto beforeCapture = HighResClock::now();
+
+            camera.capture(settings);
+
+            const auto afterCapture = HighResClock::now();
+            auto captureTime = afterCapture - beforeCapture;
+            if(captureTime < captureCycle)
+            {
+                std::this_thread::sleep_for(captureCycle - captureTime);
+            }
+            else
+            {
+                std::cout << "Your capture time is longer than your desired capture cycle."
+                          << "Please increase the desired capture cycle." << std::endl;
+            }
```

Comment:
Not sure if I like this. We say "please increase the desired capture cycle", but the loops keeps on going. If they don't change the warm up time this is going to print every 5 seconds for 10 minutes. 
Not sure if that is a problem, but just thinking out loud here.

Reply:
If we are using capture assistant we could have at a variable for max capture time and compared that to cycle time. 
But since cycle time is 5s by default and capture assistant i 1s by default it should work out-of-the-box and if they do the modification they will be aware of what is happening. 
Don't think my first comment is important..